mobileApp/src/machines/gameMachine/gameMachine.ts

```diff
@@ -0,0 +1,112 @@
+import {assign, Machine} from "xstate";
+
+import {IInitState} from "../../redux/reducers";
+import {calculateWinner} from "../../utils/calculateWinner";
+
+const initialState: IInitState = {
```

Comment:
This object is context, not a state. So, please rename it with its type without this misunderstanding.